### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by mplebanski**
Nitpicking. I found some typos in GUI when creating a blender task

### CODE REVIEW

File: src/components/tasks/TaskDetail.js [hunk 1]
```diff
@@ -1228,9 +1228,9 @@ export class TaskDetail extends React.Component {
                                         Define frames to render. You can
                                         separate frame numbers
                                         <br />
-                                        with ;, eg. 1;4;7 will define frame 1, 4
+                                        with ; e.g. 1;4;7 will define frame 1, 4
                                         and 7. You can also define frames ranges
-                                        with -{" "}
+                                        with {" "}.
```
Comment (mfranciszkiewicz) [inline src/components/tasks/TaskDetail.js]: `-` was there on purpose. Citing the documentation:

> You can also define frames ranges with -, eg. 1-4 will define frame 1, 2, 3 and 4

Comment (mplebanski) [reply src/components/tasks/TaskDetail.js]: Sorry then, I changed it because It was inconsistent with some other citings in TaskDetails.js.

Comment (mplebanski) [reply src/components/tasks/TaskDetail.js]: @mfranciszkiewicz What should I do about it? Could you suggest an edit?

Comment (mfranciszkiewicz) [reply src/components/tasks/TaskDetail.js]: I believe @mdtanrikulu should share his thoughts on this.

Comment (mdtanrikulu) [reply src/components/tasks/TaskDetail.js]: we can add double quotes to point the character. something like,  `with "-" character.` Also please add dot before `{" "}` Cuz it creates space between the following link and sentence.

**Review by mdtanrikulu — CHANGES_REQUESTED** [review]

**Pull request merged**